.github/workflows/conformance-gateway-api.yaml

```diff
@@ -16,15 +16,15 @@ on:
       extra-args:
         description: "[JSON object] Arbitrary arguments passed from the trigger comment via regex capture group. Parse with 'fromJson(inputs.extra-args).argName' in workflow."
         required: false
-        default: '{}'
```

Comment:
this seems to have extra changes from original commits, probably from auto format from IDE ?

Reply:
Can you help to revert some other changes? last time, I and Nicolas were struggling with all indentations.

Reply:
I fixed it manually and removed the formatting when saving file for the moment in VSCode.

Reply:
Thanks a lot :)